<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LampWidget</class>
 <widget class="QWidget" name="LampWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DomoticLamp1</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="LampsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnPower">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Ressources/power-button-icon-8371.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="sldIntesity">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>powerPressed()</slot>
 </slots>
</ui>
